front/src/views/Products/components/GlobalFilter.tsx

```diff
@@ -0,0 +1,47 @@
+import { useState, ChangeEventHandler } from "react";
+import { useAsyncDebounce } from "react-table";
+import { SearchIcon } from "@chakra-ui/icons";
+import {
+  Input,
+  InputGroup,
+  InputLeftElement,
+  useDisclosure,
+  useMediaQuery,
+} from "@chakra-ui/react";
+
+interface IProps {
+  globalFilter: string;
+  setGlobalFilter: (filterValue: string | undefined) => void;
+}
+
+export function GlobalFilter({ globalFilter, setGlobalFilter }: IProps) {
```

Comment:
again is this not something we can unify with the global filter from Manage Boxes and move into the components folder?